config/default-amo.js

```diff
@@ -23,4 +23,27 @@ module.exports = {
       ],
     },
   },
+  validUrlExceptions: [
```

Comment:
Could you put a comment above this saying that it needs to be kept in sync with https://github.com/mozilla/addons-server/blob/master/src/olympia/lib/settings_base.py#L262 for now?

Also, in a separate PR, could you add a comment to the addons-server settings? It should say that this list will eventually be handled in addons-frontend, maybe with a link back to this PR for now.

It's a shame that we'll have to keep them in sync.